README.md

```diff
@@ -79,7 +80,7 @@ Label, hint, and legend text is loaded from localisation using a default key str
 
 ### Other available components:
 ```
-hmpoCharsLeft(params);
+hmpoCharacterCount(ctx, params)
```

Comment:
Should be in `Available form wizard components` rather than here I think, that's about actual form fields it looks like